src/functions/template-tags/ical.php

```diff
@@ -20,31 +25,40 @@ function tribe_get_single_ical_link() {
 	/**
 	 * iCal Link
 	 *
-	 * Returns a sitewide ical link
+	 * Returns a sitewide "Export Events" iCal link
 	 *
 	 * @return string URL for ical dump.
 	 */
 	function tribe_get_ical_link() {
 		$output = tribe( 'tec.iCal' )->get_ical_link();
 
+		/**
+		 * Filters the "Export Events" iCal link.
+		 *
+		 * Please note that tribe-events.js dynamically sets the iCal link in most contexts. To
+		 * override this behavior so that a custom link from the tribe_get_ical_link filter is the
+		 * one that's always used, please also use the tribe_force_filtered_ical_link filter.
+		 *
+		 * @see tribe_force_filtered_ical_link
+		 * @param string $output The "Export Events" iCal link URL.
+		 */
 		return apply_filters( 'tribe_get_ical_link', $output );
 	}
-
 }
 
 if ( ! function_exists( 'tribe_get_gcal_link' ) ) {
 	/**
 	 * Google Calendar Link
 	 *
-	 * Returns an "add to Google Calendar link for a single event. Must be used in the loop
+	 * Returns an "add to Google Calendar link for a single event. Must be used in the loop.
```

Comment:
Since we're modifying this anyway, should we also add the closing `"`?

Reply:
good catch!